tests/pytests/test_search_params.py

```diff
@@ -747,3 +747,20 @@ def test_sortable_unf(env):
 
 def test_sortable_NOunf(env):
     unf(env, is_sortable_unf=False)
+
+
+def test_filter_on_missing_values():
+    """Tests the missing values indexing feature with the `exists` operator"""
```

Comment:
Why is this test in `test_search_params.py`?

Reply:
Moved to test_missing.py